pennylane/operation.py

```diff
@@ -421,16 +422,18 @@ class Expectation(Operation):
     * :attr:`~.Operation.grad_recipe`
 
     Args:
-        args (tuple[float, int, array, Variable]): Expectation parameters
+        args (tuple[float, int, array, Variable]): observable parameters
 
     Keyword Args:
         wires (Sequence[int]): subsystems it acts on.
             Currently, only one subsystem is supported.
         do_queue (bool): Indicates whether the operation should be immediately
             pushed into a :class:`QNode` circuit queue. This flag is useful if
```

Comment:
Don't observables end up in a separate observable queue?

Reply:
```suggestion
            pushed into a :class:`QNode` observable queue. This flag is useful if
```